structs.go

```diff
@@ -109,6 +112,28 @@ func (c *checker) checkStructs() {
 			issues.Report(c.Pass)
 		}
 	})
+
+	// This is necessary to catch generic type instantiations.
+	// Generic types have a generic TypeSpec which does not have defined type params.
+	// Because of that, the analysis done on those structs is not complete.
+	//
+	// Furthermore, the generic type does not match the concrete type obtained following the instantiation.
+	// This means that when we find calls to Configure using instantiated generic types as the argument, we do not
+	// have any available analysis information.
+	//
+	// Running the analysis on all generic type instances avoids these issues.
```

Comment:
maybe you could add a comment about the fact the code could be replaced by accessing the field directly

https://pkg.go.dev/golang.org/x/exp/typeparams#GetInstances

https://cs.opensource.google/go/x/exp/+/e7e105de:typeparams/typeparams_go118.go;l=132

Reply:
Replaced the call to `GetInstances` with a direct access to `TypesInfo.Instances`